Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -477,7 +477,13 @@ void HexEditor::keydown_event(GUI::KeyEvent& event)
         move_and_update_cursor_by(-m_position);
         return;
     }
-    
```

Comment:
While you are rebasing, if you can figure out what happened to this line, it would be great!
Using a formatter will help you with that, we have a documentation to run them automatically on each commit [here](https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md#commit-hooks).
If you set up that correctly, you will also get rid of BuggieBot messages in the future.